### DISCUSSION TITLE
Rule matching problem

### DISCUSSION THREAD

**Discussion by Evilglory1371** [corazawaf/coraza#955] [Q&A]
I retrieve request and response data from the database and put them into WAF to match the rules, but once I match one alarm, I no longer match the rules downwards https://github.com/corazawaf/coraza/discussions What should I do if the multiple matching mentioned in it is used to modify the rules, but it does not achieve the effect of matching a data to an alarm and continuing to match other data downwards?

**Comment by Venous-W** [comment]
发一下你的配置文件看看

**Reply by jptosso** [reply]
Please note that the project language is english. Other languages won't be answered and will be flagged for deletion.
 We understand that you are interested in contributing, but please to keep everything consistent, use english.

**Comment by jptosso** [comment]
Rules won't keep matching if the request gets interrupted with a disruptive action like Deny
